model-engine/model_engine_server/infra/gateways/resources/k8s_endpoint_resource_delegate.py

```diff
@@ -61,6 +61,13 @@
 BASE_PATH_IN_ENDPOINT = "/app"
 
 DATADOG_ENV_VAR = {"DD_TRACE_ENABLED", "DD_SERVICE", "DD_ENV", "DD_VERSION", "DD_AGENT_HOST"}
+LWS_DEFAULT_ENV_VAR = {
+    "K8S_OWN_POD_NAME",
```

Comment:
what does OWN stand for here

Reply:
it's basically synonymous with "SELF", i.e. the value of that env var is the pod's own name, e.g. in a pod named `pod1` the value of the env var is `pod1`